timereport/lib/list.py

```diff
@@ -14,12 +36,15 @@ def get_between_date(url, start_date, auth_token, end_date):
     :return:
     """
     params = {'access_token': auth_token, 'startDate': start_date, 'endDate': end_date}
-    res = requests.get(url=url, params=params)
-    print(f'Response code is: {res.status_code}')
-    print(f'Url is: {res.url}')
-    for resp in res.json():
-        print(f'{resp}')
-    if res.status_code == 200:
-        return True
-    else:
-        return False
+    try:
+        res = requests.get(url=url, params=params)
```

Comment:
It's only this that will cause the exception right? I prefer to have smaller blocks (relevant code) in the `try/except` to minimize risk of catching the wrong things. Especially since you have a bare `except` it can make it very hard to debug